include/megaapi.h

```diff
@@ -8507,12 +8507,15 @@ class MegaApi
          * The associated request type with this request is MegaRequest::TYPE_ACCOUNT_DETAILS
          *
          * Use this version of the function to get just the details you need, to minimise server load
-         * and keep the system highly available for all.
+         * and keep the system highly available for all.  At least one flag must be set.
```

Comment:
In case none of the flags is set, the request should fail with `API_EARGS`, not silently ignore the error.

Reply:
Done!